frontends/web/src/components/backups/create.jsx

```diff
@@ -74,6 +75,10 @@ export default class Create extends Component {
                 alert(data.errorMessage); // eslint-disable-line no-alert
             } else {
                 this.props.onCreate();
+                console.log(data);
+                if (!data.verification) {
+                    alert(this.props.t('backup.create.verificationFailed'));
+                }
```

Comment:
A bit unusual... I don't think the app uses native alerts anywhere else. Maybe change to <Message> or whatever is used normally, for consistency?

Reply:
Uses it in lots of places.

Reply:
Afaik all dialogs and single-button actions use alerts, and the setup wizard uses messages.